### PR TITLE
Ask VK for a bigger photo

### PR DESCRIPTION
**Pull request by bassx1**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "name": "socialiteproviders/vkontakte",
+    "name": "bassx1/vkontakte",
```
Comment (jhaoda) [inline composer.json:2]: WTF?

File: composer.json [hunk 2]
```diff
@@ -8,7 +8,7 @@
     }],
     "require": {
         "php": "^5.6 || ^7.0",
-        "socialiteproviders/manager": "~2.0 || ~3.0"
```
Comment (jhaoda) [inline composer.json:11]: WTF?

**Review by jhaoda — CHANGES_REQUESTED** [review]

**Comment by faustbrian** [comment]
Yeah no that won't be merged... Changing yourself to the author of the package and removing support for the current version is just... I don't even know.

**Comment by bassx1** [comment]
Oh guys, sorry about that, I made it to upload my forked repo to packagist and use it temporarily. 
I wanted to propose only [ first commit](https://github.com/SocialiteProviders/VKontakte/pull/24/commits/ab49b7a66b8c36c463557a668493cc2ab635c61b) for pull request.

**Pull request closed without merging**